components/edit-listing-form.tsx

```
@@ -0,0 +1,618 @@
+          <Card>
+            <CardContent className="p-6">
+              <div className="grid gap-6">
+                <div className="flex flex-col gap-2">
+                  <Label>Property Photos</Label>
+                  <p className="text-sm text-muted-foreground mb-4">
+                    Upload high-quality photos of your property. The first image will be used as the main photo.
+                  </p>
+
+                  <div className="grid grid-cols-2 md:grid-cols-4 gap-4">
+                    {images.map((image, index) => (
+                      <div key={index} className="relative group">
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)